src/Cabal2nix.hs

```diff
@@ -0,0 +1,176 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards   #-}
+
+module Cabal2nix (Options (..), main, mainWithArgs) where
+
+
+import Control.Exception ( bracket )
+import Control.Lens
+import Data.Maybe ( fromMaybe )
+import Data.Monoid ( (<>) )
+import qualified Data.Set as Set
+import Data.String
+import qualified Distribution.Compat.ReadP as P
+import Distribution.Compiler
+import Distribution.Nixpkgs.Fetch
+import Distribution.Nixpkgs.Haskell
+import Distribution.Nixpkgs.Haskell.FromCabal
+import Distribution.Nixpkgs.Haskell.PackageSourceSpec
+import Distribution.Nixpkgs.Meta
+import Distribution.PackageDescription ( FlagName (..), FlagAssignment )
+import Distribution.Simple.Utils ( lowercase )
+import Distribution.System
+import Distribution.Text
+import Language.Nix
+import Options.Applicative
+import Paths_cabal2nix ( version )
+import System.Environment ( getArgs )
+import System.IO ( hFlush, stderr, stdout )
+import qualified Text.PrettyPrint.ANSI.Leijen as P2 hiding ( (<$>), (<>) )
+import Text.PrettyPrint.HughesPJClass ( Doc, Pretty (..), text, vcat, hcat, semi )
+
+
+data Options = Options
+  { optSha256                    :: Maybe String
```

Comment:
I agree cosmetic changes should go in separate commits. The change was introduced because I had `stylish-haskell` running automatically, and after moving the code around I missed some of the changes it made. I'll be more careful about that next time.